src/PromClientRegistryConfig.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * © 2023 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import fs from 'fs'
+import client from 'prom-client'
+
+const metricsMap = [
+  {
+    name: 'json_schema_version_info',
+    packageName: 'json-schema',
+    help: 'The json-schema version by package.json',
```

Comment:
Let's replace "json-schema" with "@manuscripts/json-schema" in the help. Same for the rest.